source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql

```diff
@@ -1,17 +1,16 @@
 ﻿CREATE TABLE [dbo].[UnmatchedSpecimensLinkedNotifications]
 (
-	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL PRIMARY KEY,
+	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL,
```

Comment:
Should we be replacing this with a primary key on the `ReferenceLaboratoryNumber` and the `NotificationID`?

Perhaps we shouldn't, as it's at least conceptually possible that the same notification could be related to a lab result in two different ways